test/rewards/AutoCompoundingRewards.ts

```diff
@@ -686,6 +716,85 @@ describe('AutoCompoundingRewards', () => {
                     }
                 });
 
+                describe('auto-process rewards', () => {
+                    const AUTO_PROCESS_COUNT = 3;
+
+                    const setups = [
```

Comment:
can have an array of token symbols and initialize the rest in the loop

Reply:
These are just random amounts that I've picked.
Initializing them in a loop means that we will not be able to easily change or add more of them.

Reply:
ok, though look at the numbers now

Reply:
Yeah, I wasn't picking "that randomly". But the idea is that they are independent of each other.
Computing them in a loop would some dependency which is not required (nor desired) in this case.

Reply:
sounds good